advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -32,7 +32,7 @@
    "outputs": [],
```

Comment:
<p>Based on the CI log, this seems failing:</p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
Exception encountered at "In [16]":
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
&lt;ipython-input-16-e17e3fc514f0&gt; in &lt;module&gt;
      3 # Check output
      4 threshold = 0.5
----&gt; 5 y_results = np.array(['yes' if float(i) &gt; threshold else 'no' for i in results])
      6 
      7 print(Counter(y_results))

&lt;ipython-input-16-e17e3fc514f0&gt; in &lt;listcomp&gt;(.0)
      3 # Check output
      4 threshold = 0.5
----&gt; 5 y_results = np.array(['yes' if float(i) &gt; threshold else 'no' for i in results])
      6 
      7 print(Counter(y_results))

ValueError: could not convert string to float: 'no'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>